docs/openapi.yml

```diff
@@ -102,6 +105,9 @@ paths:
     parameters:
       - $ref: '#/components/parameters/namespace'
     get:
+      parameters:
```

Comment:
_Minor:_ The namespace endpoint doesn't currently support `limit` and `offset`, but I'll be working on introducing that functionality in the next few days